test/integration/product-import.spec.coffee

```diff
@@ -189,6 +189,36 @@ describe 'Product Importer integration tests', ->
     .then (result) ->
       service.byId(id).delete(result.body.version)
 
+  it 'should handle an empty import', (done) ->
```

Comment:
i am not sure if it matches issue description. You still do unnecessary request with no skus provided?